democracy/migrations/0030_migrate_translatable_fields.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.10 on 2016-12-08 12:00
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+from django.core.exceptions import ObjectDoesNotExist
+
+IMPORT_LANGUAGE_CODE = 'fi'  # Assume that the existing content is in Finnish
+
+
+def assign(obj_dest, obj_src, fields):
+    for field in fields:
+        if field in ['id', 'master', 'language_code']:
+            continue
+        setattr(obj_dest, field, getattr(obj_src, field))
+
+
+def forwards_func_factory(modelname):
+    def forwards_func(apps, schema_editor):
+        MyModel = apps.get_model('democracy', modelname)
+        MyModelTranslation = apps.get_model('democracy', '%sTranslation' % modelname)
+        model_fields = [f.name for f in MyModelTranslation._meta.get_fields()]
+
+        for object in MyModel.objects.all():
+            translation = MyModelTranslation(
+                master_id=object.pk,
+                language_code=IMPORT_LANGUAGE_CODE,
+            )
+            assign(translation, object, model_fields)
+            translation.save()
+    return forwards_func
+
+
+def backwards_func_factory(modelname):
+    def backwards_func(apps, schema_editor):
+        MyModel = apps.get_model('democracy', modelname)
+        MyModelTranslation = apps.get_model('democracy', '%sTranslation' % modelname)
+        model_fields = [f.name for f in MyModelTranslation._meta.get_fields()]
+
+        for object in MyModel.objects.all():
+            try:
+                translation = _get_translation(object, MyModelTranslation)
+                assign(object, translation, model_fields)
+                object.save()   # Note this only calls Model.save() in South.
+            except:
```

Comment:
This hides all possible errors, wouldn't it be better to catch just expected exceptions?